src/vivarium_nih_us_cvd/constants/metadata.py

```diff
@@ -11,14 +11,63 @@
 CLUSTER_PROJECT = "proj_simscience_prod"
 
 CLUSTER_QUEUE = "all.q"
-MAKE_ARTIFACT_MEM = "10G"
-MAKE_ARTIFACT_CPU = "1"
-MAKE_ARTIFACT_RUNTIME = "3:00:00"
+MAKE_ARTIFACT_MEM = 10  # GB
+MAKE_ARTIFACT_CPU = 1
+MAKE_ARTIFACT_RUNTIME = "6:00:00"
```

Comment:
Is 6 hours needed?

Reply:
my data point of 1 for Alabama took 3.6 hours. I'm not sure how much overhead to give it by default. I'll change to 4 for now.